docs/docs/lib/edge/cloudflare.mdx

```diff
@@ -0,0 +1,351 @@
+RUN_CROSS_ORIGIN_URL_REDIRECT_EXPERIMENTS="everywhere"|"edge"|"browser"|"skip"  # default: "browser"
+# Mutate browser URL via window.history.replaceState() to reflect the new URL:
+INJECT_REDIRECT_URL_SCRIPT="true"  # default "true".
+
+# Do not inject a bootstrapped JavaScript SDK onto the page:
+# ==========
+DISABLE_INJECTIONS="true"  # default "false"
+
+
+# Customize the edge or injected browser SDK behavior:
+# ==========
+ENABLE_STREAMING="true"  # default "false". Streaming SSE updates on browser.
+ENABLE_STICKY_BUCKETING="true"  # default "false". Use cookie-based sticky bucketing on edge and browser.
+```
+
+#### Runtime configuration
+
+You may want to provide context to your edge app at runtime rather than using environment variables. For example, if you have additional [targeting attributes](/lib/js#attributes) available, you may inject them by modifying your request handler code:
+
+```ts
+import { handleRequest } from "@growthbook/edge-cloudflare";
+import { parse } from "cookie";
+
+export default {
+  fetch: async function (request, env, ctx) {
+    const cookie = parse(request.headers.get("Cookie") || "");
+    const config = {
+      attributes: {
+        userType: cookie["userId"] ? "logged in" : "anonymous"
+      }
+    };
+    return await handleRequest(request, env, config);
+  },
+};
+```
+
+#### More customization options
+
+For a full list of customizations, view our vendor-agnostic [Edge Utility repository <ExternalLink />](https://github.com/growthbook/growthbook-proxy/tree/main/packages/lib/edge-utils).
+
+### Set up a Payload Cache
+
+You can configure GrowthBook payload caching by using a [CloudFlare KV <ExternalLink />](https://developers.cloudflare.com/kv/reference/how-kv-works/) store. This eliminates network requests from your edge to GrowthBook which speeds up page delivery while reducing network costs.
+
+Our CloudFlare Edge App will automatically use either webhook-based or just-in-time payload caching (or both) depending on how you've set up your KV namespaces, bindings, and SDK Webhooks.
+
+More information about setting up your payload cache can be found in the [Payload Caching with CloudFlare KV Store](#payload-caching-with-cloudflare-kv-store) doc section below.
+
+### Tracking Experiment Views
+
+Running A/B tests requires a [tracking callback](/lib/js#experimentation-ab-testing). Our turnkey Edge App defaults to using built-in front-end tracking. The tracking call automatically integrates with Segment.io, GA4, and Google Tag Manager by using the mechanism outlined in our [HTML Script Tag](/lib/script-tag#tracking-experiment-views). In order to do this, the app keeps track of tracking calls triggered on edge and injects them into the front-end SDK to be automatically triggered on page load.
+
+You may wish to either customize front-end tracking or switch to edge tracking (or use both concurrently if running hybrid edge + front-end experiments).
```

Comment:
```suggestion
You may wish to either customize front-end tracking or switch to edge tracking (or use both concurrently if running hybrid edge + front-end experiments). While edge tracking requires more custom configuration, it provides you the control to ensure tracking calls are triggered before there is any potential deviation in user experience due to an experiment. This is the safest way to ensure balanced traffic in experiments (by ensuring no experiment traffic is lost before the tracking callback fires).
```

Feel free to ignore, or just delete the last sentence of my suggestion. I just feel we should educate about this whenever possible.